pillar/coalition.sls

```diff
@@ -0,0 +1,39 @@
+network:
+  host_id: ocp21
+  ipv4: 139.162.211.65
+  networkd:
+    template: linode
+    gateway4: 139.162.211.1
+
+apache:
+  public_access: True
+  sites:
+    coalition:
+      configuration: wordpress
+      servername: www.open-spending.eu
+      serveraliases: ['open-spending.eu']
+      context:
+        user: coalition
+        socket: /var/run/php/php-fpm-coalition.sock
+
+php:
+  version: '8.1'
```

Comment:
Why not 8.2?

Reply:
`8.1` is the version provided and managed directly from Ubuntu for `22.04`.
This means that Canonical will continue to release security patches until the operating systems end of life date (2027), even after PHP's end of life date (Dec 2025).

Another consideration is that WordPress is still being updated for PHP `8.x` support. Officially running WordPress on any PHP `8.x` version is in "Beta" support so there may be bugs/issues using the latest version.

Reply:
If / when we do want to upgrade PHP, we can install the Ondrej PHP repository.
https://launchpad.net/~ondrej/+archive/ubuntu/php

Reply:
Sounds good. I've now mentioned this in the docs.